data/Language/Flow.yaml

```diff
@@ -0,0 +1,50 @@
+identifier: Flow
+full_name: Flow
+wikidata: Q59413589
+creation_date: 2014
+extensions:
+  - js.flow
+language_derivative: JavaScript
```

Comment:
Should be a list? See Netlify deploy error https://app.netlify.com/sites/coast/deploys/5c125653227091000a66a819
```
8:54:09 PM: 2018-12-13 12:54:01,797	statik.project	ERROR	Model error: For model "Language", field "language_derivative" should be a list. (in file "/opt/build/repo/data/Language/Flow.yaml").
```